### DISCUSSION TITLE
JAVA_OPTIONS configuration on SNAP installation

### DISCUSSION THREAD

**Discussion by leinad87** [dbeaver/dbeaver#14616] [Q&A] [answered]
I use to modify `java.security` file to modify SSL/certificate valid cyphers. When dbeaver is installed with SNAP, this file is located in `/snap/dbeaver-ce/current/usr/share/dbeaver-ce/jre/conf/security/java.security`, however this is a read-only mount path. 

Another option could be use java parameter `-Djava.security.properties=<path_to_java_security_file>`. 

How can I solve this?

**Comment by Matvey16** [comment, marked as answer]
@leinad87 Unfortunately it is how snap works, so the only options are to either use java parameter in command line or to manually install Dbeaver with a package from https://dbeaver.io/download/
